Cargo.toml

```diff
@@ -43,7 +43,7 @@ byteorder = { version = "1.3.4", default-features = false }
 cargo_metadata = { version = "0.12.0", default-features = false }
 cfg-if = { version = "1", default-features = false }
 chrono = { version = "0.4", default-features = false }
-clap = { version = "3.0.14", default-features = false, features = ["derive"] }
+clap = { version = "3.0.14", default-features = false, features = ["std", "derive"] }
```

Comment:
Passing default-features=false for the crates used by the std build system is kinda weird, if we removed that you wouldn't need std here.

Reply:
After chatting in #oxide-hubris, I decided to leave this as-is; we have the build system and embedded crates all jumbled up, so it's easier to be consistent if we _always_ set `default-features = false`